articles/virtual-machines/windows/run-command-managed.md

```diff
@@ -254,28 +255,29 @@ PUT /subscriptions/<subscriptionId>/resourceGroups/<resourceGroupName>/providers
 ### Notes
  
 - You can provide an inline script, a script URI, or a built-in script [command ID](run-command.md#available-commands) as the input source. Script URI is either storage blob SAS URI with read access or public URI.
-- Only one type of source input is supported for one command execution.  
+- Only one type of source input is supported for one command execution.
+- Starting with API version 2023-03-01, you can set the property `treatFailureAsDeploymentFailure` to **true** causing the deployment to fail when there is a failure in the script. If set to **false**, ProvisioningState would only reflect whether the run command was run or not by the extensions platform, it would not indicate whether script failed in case of script failures. 
```

Comment:
```suggestion
- Starting with API version 2023-03-01, you can set the property `treatFailureAsDeploymentFailure` to **true** causing the deployment to fail when there is a failure in the script. If set to **false**, ProvisioningState would only reflect whether the run command was run or not by the extensions platform. It would not indicate whether the script failed in case of script failures. 
```